src/NodeVisitor/EscaperNodeVisitor.php

```diff
@@ -67,7 +67,7 @@ public function enterNode(Node $node, Environment $env): Node
         return $node;
     }
 
-    public function leaveNode(Node $node, Environment $env): ?Node
```

Comment:
PHPStan report the null type as useless.

Since the class is final we can remove it.